packages/ses/test/test-make-eval-function.js

```diff
@@ -21,7 +21,7 @@ test('makeEvalFunction - leak', t => {
   t.is(safeEval('none'), undefined);
   t.is(safeEval('this.none'), undefined);
 
-  safeEval('none = 8');
+  safeEval('this.none = 8');
 
   t.is(safeEval('none'), 8);
   t.is(safeEval('this.none'), 8);
```

Comment:
If assignment to the global as a free variable that is an existing property of the global object isn’t supposed to work, these three lines can probably be deleted since they’re redundant with the next three.

@erights Was this test valid previously? @kumavis Was this changed because the refactor made this behavior throw?

Reply:
line 24 was indeed changed because in this version it throws. it is a sloppyGlobals assignment. the previous version accidently allowed sloppyGlobals if the property was on the realm globalThis.

this change in behavior is recorded in the PR top comment